zend_abstract_interface/config/config.c

```diff
@@ -10,7 +10,7 @@
 HashTable zai_config_name_map = {0};
 
 #ifndef _WIN32
-_Static_assert(ZAI_CONFIG_ENTRIES_COUNT_MAX < 256, "zai config entry count is overflowing uint8_t");
+_Static_assert(ZAI_CONFIG_ENTRIES_COUNT_MAX < 258, "zai config entry count is overflowing uint8_t");
```

Comment:
Did ... you actually possibly consider that this could be a _problem_? Like uint8_t must not be larger than 256.
I actually fix this on another branch though (in #3115).
I'll make sure to merge that one first.

Reply:
I was getting errors related to adding the two new configs and thought this was the fix...